tests/assembleCoaddTestUtils.py

```diff
@@ -46,38 +46,20 @@
 __all__ = ["MockWarpReference", "makeMockSkyInfo", "MockCoaddTestData"]
 
 
-class MockWarpReference(lsst.daf.butler.DeferredDatasetHandle):
+class MockWarpReference(pipeBase.InMemoryDatasetHandle):
     """Very simple object that looks like a Gen 3 data reference to a warped
     exposure.
 
     Parameters
     ----------
     exposure : `lsst.afw.image.Exposure`
```

Comment:
I think this should at least be documented as mapping to the `inMemoryDataset` attribute of the dataclass.